roles/ci_nmstate/tasks/nmstate_unmanaged_provision_node.yml

```diff
@@ -21,6 +21,17 @@
     state: latest  # noqa: package-latest
     use: "{{ hostvars[cifmw_ci_nmstate_unmanaged_host]['ansible_facts']['pkg_mgr'] }}"
 
+#TODO(hjensas): I don't think the cifmw_ci_nmstate_instance_config "direct pass trough"
```

Comment:
@pablintino - I initially tried to use `cifmw_ci_nmstate_instance_config` - but it failed.

```
TASK [ci_nmstate : Build NMstates from the Networking Environment Definition] ***
fatal: [instance]: FAILED! =>
  msg: 'An unhandled exception occurred while templating ''{{ cifmw_networking_env_definition | ci_nmstate_map_instance_nets(item) }}''. Error was a <class ''KeyError''>, original message: ''parent_interface'''
```

I think that never worked, because we don't load net-env def when `cifmw_ci_nmstate_instance_config` is set?

Reply:
Marking as resolved, ref: https://github.com/openstack-k8s-operators/ci-framework/pull/1995